<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Salas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableSalas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>531</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Capacidade</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnIncluirSala">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Incluir Sala</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAlterarSala">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>560</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Alterar Sala</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExcluirSala">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>560</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir Sala</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
